processes/third_party_vuln_process.md

```diff
@@ -125,6 +125,7 @@ by PRing their acceptance to this file:
 * @lirantal - Liran Tal
 * @MarcinHoppe - Marcin Hoppe
 * @vdeturckheim - Vladimir de Turckheim
+* @pxlpnk - Andreas Tiefenthaler
```

Comment:
We usually put the names in alphabetic order.

Reply:
fixed